client/web/src/repo/blame/useBlameHunks.ts

```diff
@@ -74,65 +101,151 @@ const fetchBlame = memoizeObservable(
             { repo: repoName, rev: revision, path: filePath }
         ).pipe(
             map(dataOrThrowErrors),
-            map(({ repository }) => repository?.commit?.blob?.blame)
+            map(({ repository }) => repository?.commit?.blob?.blame),
+            map(blames => (blames ? blames.map(blame => addDisplayInfoForHunk(blame, sourcegraphURL)) : undefined)),
+            map(blames => ({ current: blames }))
```

Comment:
```suggestion
            map(hunks => (hunks ? hunks.map(blame => addDisplayInfoForHunk(hunk, sourcegraphURL)) : undefined)),
            map(hunks => ({ current: hunks }))
```

nit: just to be consistent with the type name from the return value (`BlameHunk`) and the formatting function (`addDisplayInfoForHunk`)